root/var/www/html/config.inc.php

```diff
@@ -4,38 +4,38 @@
  * Database settings
  ******************/
 // Which database system: "pgsql"=PostgreSQL, "mysql"=MySQL
-$dbsys = "mysql";
+$dbsys = $_ENV['MRBS_DB_SYSTEM'] ?? 'mysql';
```

Comment:
Default values for database-related things don't make sense here. If the variable is not set, the `50-config` script will fail, and we will never get to this point.

If you want to set a default value, please do so in the `50-config` script :)